src/commands/address.js

```diff
@@ -7,15 +8,14 @@ function address (message, dogecoinNode) {
 
   // Will create a new account if doesn't exist... ? Should we allow this ?
   // Yes
-  dogecoinNode.getAccountAddress(account, function (err, address) {
-    if (err) {
+  getAccountAddress(account)
```

Comment:
Le paramètres dogecoinNode  de la fonction address (ligne 6) est encore utilisé ?

Reply:
No good catch it should not be used anymore.